project_euler/problem_050/sol1.py

```diff
@@ -0,0 +1,80 @@
+"""
+Project Euler Problem [50]: https://projecteuler.net/problem=50
+    
+
+The prime 41, can be written as the sum of six consecutive primes:
+41 = 2 + 3 + 5 + 7 + 11 + 13
+
+This is the longest sum of consecutive primes that adds to a prime below one-hundred.
+
+The longest sum of consecutive primes below one-thousand that adds to a
+ prime, contains 21 terms, and is equal to 953.
+
+Which prime, below one-million, can be written as the sum of the 
+most consecutive primes?
+"""
+
+
+def is_prime(number: int) -> [bool]:

```

Comment:
```suggestion
def is_prime(number: int) -> bool:
```
brackets are not required